source/scripts/script.js

```diff
@@ -1,10 +1,5 @@
-const API_KEY = '8aaa6b0816db4a99b92e7852d125a9aa';
-// API_KEY4 (Nhi): 8aaa6b0816db4a99b92e7852d125a9aa
-// API_KEY3 (Nhi): c8f83bb3a9af4355b12de10250b24c88
-// API_KEY2 (Nhi): fafd5e810c304ed3b4f9984672cb21ee
-// API_KEY1: 4d936c811cda46879d4749def6bb36a1
-const url = `https://api.spoonacular.com/recipes/complexSearch?apiKey=${API_KEY}&addRecipeInformation=true&instructionsRequired=true`;
-const recipes = [];
+// Import API from file
+import { fetchRecipes } from '/source/scripts/api_script.js';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group23/cse110-fa21-group23/pullRequest?prid=8408391)
```suggestion
import { fetchRecipes } from "/source/scripts/api_script.js";
```